media-sound/carla/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST carla-2.1.1.tar.gz 9177989 BLAKE2B 87f5fe97e3d3e4d7f25da4b14464069b6cd1c83405c458b9d80add3e54d61d59763f074a3a85c9c88efdcd8f5790303992cc3233b767189cd4f50c60cbeda892 SHA512 e8da99d7bddff1415dd54ac3e1abe90fbccb42877d2daec0a60ad046e6d15531ebf8328c4d44c06cb14a7a5308d43be62d88eed6268a55dfbfae0f620a5f04dc
 DIST carla-2.2.0.tar.gz 9844747 BLAKE2B 1fd778a1c2904fe982c3aec32e9bb39fff679bea0fe8d1fe41d10bbc5347dd82a6f3dcd33d19aee8bc3b123e387a8b7655c9735fca784d642596f0df9085e38e SHA512 afb2beab979f801817a7fd7663bfdc90800db021679243faa5acb15de60f6dc281032e05c51a8625124c69e7759099fc682678eb98c08507f952763e665ac9c4
```

Comment:
We can just drop the old versions, generally we keep/have kept live, latest and latest-1 (where -1 is mostly the major or minor version)

Reply:
Well, that wasn't the initial purpose of this PR, but I'll do that. I don't see the need for keeping more than two versions around, anyway.

Reply:
dropped 2.1.1